README.md

```diff
@@ -1,24 +1,137 @@
+
+- **Ruby on Rails**: You'll need the Ruby on Rails framework installed. If you don't have it yet, you can install it using the following command:
+  ```
+  gem install rails
+  ```
+
+- **Code Editor**: You should have a code editor like Visual Studio Code (VSCode) installed. You can download VSCode from the [official website](https://code.visualstudio.com/).
+
+- **Git**: Git is essential for version control. Make sure you have Git installed and configured on your system. You can download Git from the [official Git website](https://git-scm.com/downloads).
+
+- **PostgreSQL**: Ensure that PostgreSQL is installed and properly configured on the server where you are developing the project. This project assumes the use of PostgreSQL as the default database. You can download PostgreSQL from the [official PostgreSQL website](https://www.postgresql.org/download/).
+
+- **Ruby on Rails Configuration**: Make sure your Ruby on Rails project is configured to use PostgreSQL as the default database.
+
+- **Node.js**: This is mandatory to run the front-end side of the application made with React
+
+With these prerequisites and configurations in place, you'll be ready to set up and run the app.
+
+### 📖 Usage <a name="usage"></a>
+Follow these steps to use the app:
+
+### Backend with Ruby on Rails:
+
+Open your terminal.
+
+Navigate to the directory where the project files are located:
+
+Copy code
+```bash
+cd /path/to/your/project/files
+```
+Start the Rails server:
+
+Copy code
+```bash
+rails server
+```
+### Front-end with React
+In your terminal and the root of the project.
+
+Run this for installing the dependencies:
+
+```bash
+npm install
+```
+
+And this to start the React application on the local server:
+
+```bash
+npm start
+```
+
+**Congratulations, now you have both applications running!**
+
+Now just open your browser and go to [this link](localhost:3000) and see them in action.
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Gumaro Monroy Vazquez**
+- GitHub: [GitHub](https://github.com/Movgto)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/gumaro-monroy-vazquez-1705aa165/)
+- WellFound: [Wellfound](https://wellfound.com/u/maro-monroy)
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](https://github.com/Movgto/react-rails-greetings/issues) and contribute to the project.
+
+## ⭐️ Show Your Support <a name="support"></a>
+If you like this project, please show your support with a ⭐️, that would be really meaninful for me, thank you.
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to express my gratitude to [Microverse](https://github.com/microverseinc) for providing us with the opportunity to work on this project.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📜 License <a name="license"></a>
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] 👏👏 Great job! It's a good practice to include proper licensing information for your code. However, I noticed that your readme file, the license file URL not working.  Please update your readme file with the license file link.
![image](https://github.com/Movgto/rails-react-greetings/assets/55840999/3099168a-9f38-48d5-89d9-a48c01ad0b6f)

By providing clear instructions in the README, you will ensure that others can easily use your project and understand any license needed. Keep up the great work, and happy coding! 🚀💻🌟